vmprof/reader.py

```diff
@@ -43,12 +71,12 @@ class JittedCode(int):
     pass
 
 def wrap_kind(kind, pc):
-    if kind == VMPROF_ASSEMBLER_TAG:
-        return AssemblerCode(pc)
-    elif kind == VMPROF_JITTED_TAG:
-        return JittedCode(pc)
-    assert kind == VMPROF_CODE_TAG
-    return pc
+    cls = {
+        VMPROF_ASSEMBLER_TAG: AssemblerCode,
+        VMPROF_JITTED_TAG: JittedCode,
+        VMPROF_CODE_TAG: lambda x: x,
+    }[kind]
```

Comment:
OK, I prefer the dict thing because it makes it very clear that `wrap_kind` is a `pc -> something` map, but I'm OK with reverting the change.